config/routes.rb

```diff
@@ -219,9 +219,11 @@
           post 'authenticate', on: :member
           post 'remind', on: :member
           post 'unblock_monitor', on: :member
+          get :live_feedback_settings, on: :member
           get :requirements, on: :member
           get :statistics, on: :member
           get :monitoring, on: :member
+          patch :update_live_feedback_settings, on: :member
```

Comment:
GET and PATCH are different HTTP request types, and should use the same route `:live_feedback_settings`, don't make a new route. See other routes for examples.

Reply:
done